ivy/functional/ivy/elementwise.py

```diff
@@ -5450,9 +5450,9 @@ def rad2deg(
     return ivy.current_backend(x).rad2deg(x, out=out)
 
 
-@inputs_to_ivy_arrays
 @handle_array_function
 @handle_out_argument
```

Comment:
I think `inputs_to_ivy_arrays` should be between `handle_array_function` and `handle_out_argument` based on the decorator sequence in the [`func_wrapper`](https://github.com/unifyai/ivy/blob/c63f7759b6ff6d197d4a5f21272b669d7d19289d/ivy/func_wrapper.py#L14)

Reply:
Oops, my bad. Will update this, thanks for pointing it out! :)

Reply:
Updated.